libs/sdk-backend-tiger/src/backend/workspace/catalog/datasetLoader.ts

```diff
@@ -48,12 +49,12 @@ function getAttributeLabels(
 }
 
 function isGeoLabel(label: JsonApiLabelOutWithLinks): boolean {
-    /*
-     * TODO: TIGER-HACK this is temporary way to identify labels with geo pushpin; normally this should be done
-     *  using some indicator on the metadata object. for sakes of speed & after agreement with tiger team
-     *  falling back to use of id convention.
-     */
-    return label.id.search(/^.*\.geo__/) > -1;
+    const type = label.attributes?.valueType;
+
+    return (
+        type === JsonApiLabelOutAttributesValueTypeEnum.GEO_LATITUDE ||
+        type === JsonApiLabelOutAttributesValueTypeEnum.GEO_LONGITUDE
+    );
```

Comment:
Should we check also for GEO_PUSHPIN? or should it be removed from the LabelTypeConverter?

Reply:
Checking with Tiger team, if JsonApiLabelOutAttributesValueTypeEnum should not be without GEO_PUSHPIN. So far I will keep it everywhere